src/library/masq.js

```diff
@@ -69,7 +70,7 @@ class Masq {
       db.on('ready', () => this._startReplicate(db))
     })
 
-    const profile = (await this.profileDB.getAsync('/')).value
+    const profile = await this.get('/')
     return profile
   }
 
```

Comment:
You forgot to clear `this.masterKey` in `closeProfile`

Reply:
Good ! done.